inputs/orchestrator_inner:6.json

```diff
@@ -88,6 +88,9 @@
         "insecure": true
       }
     },
+    {
+      "name": "verify_media_types"
```

Comment:
Let's add `"required": false` so it doesn't break older versions of atomic-reactor.